spec/spec_helper.rb

```diff
@@ -35,7 +35,7 @@
       end
     end
     enable_coverage :branch
-    minimum_coverage line: 99.98, branch: 94.72
+    minimum_coverage line: 99.98, branch: 93.8
```

Comment:
This seems to have snuck in

Reply:
huh. the thing i did was
`git checkout main -- spec/spec_helper.rb`
which i expected to put it back to the default version (to undo the thing commented on upthread). any idea what's up with that?

Reply:
That should! Possibly your local version of main was outdated, if we changed that recently? `git fetch` + `git checkout origin/main -- spec/spec_helper.r` should avoid that, i think

Reply:
hmmm i have done those things and it's telling me nothing changed. baffling